roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -3,6 +3,9 @@
 - set_fact:
     l_insecure_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(openshift.docker.insecure_registries)) }}"
   when: openshift.docker.insecure_registries
+- set_fact:
+    l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(openshift.docker.additional_registries)) }}"
+  when: openshift.docker.additional_registries
```

Comment:
Well, that could still break someone, Docker falls back, so it'd say let's start by having feature parity with it and add the fall back. We could revisit this later as nobody is going to really use this now. If we have people who relies on the fallback they'll get broken by this, let's have happy people testing openshift :)